### DISCUSSION TITLE
If I am a philatelist, what do I collect?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#630] [Galaxy Trivia] [answered]
**Bonus:** What is the name of the enthusiast who collects vinyl records?

**Comment by peterbaccaro** [comment, marked as answer]
If you are a philatelist, you collect **postage stamps**. Philately is the study of postage stamps and postal history, and philatelists, or stamp collectors, collect stamps from different countries and time periods. Some collectors focus on specific themes, such as animals, famous people, or historical events, while others collect stamps based on their rarity, age, or other unique characteristics.

The name of the enthusiast who collects vinyl records is a vinyl collector or a record collector.
A less common word for a record collector is **discophile**.

**Comment by ParanoidUser** [comment]
Hey @DevElkami / @DelrosarioRovic, we're looking for a word to call a person who collects vinyl records. Any ideas? 📻🎶
